public/app/features/explore/state/explorePane.ts

```diff
@@ -214,6 +216,13 @@ export const paneReducer = (state: ExploreItemState = makeExplorePaneState(), ac
     };
   }
 
+  if (contentOutlineAction.match(action)) {
+    return {
+      ...state,
+      contentOutlineVisible: action.payload.visible,
+    };
+  }
```

Comment:
imho having this state in redux is an overkill. we could have a boolean state and callback in `Explore.tsx` that would make implementation way simpler